Source/SwiftLintBuiltInRules/Rules/Lint/PrivateSwiftUIStatePropertyRuleExamples.swift

```diff
@@ -0,0 +1,247 @@
+        }
+        """),
+        Example("""
+        struct MyScene: Scene {
+            @StateObject ↓var model = DataModel()
+        }
+        """),
+        Example("""
+        struct ContentView: View {
+            @StateObject ↓var model = DataModel()
+        }
+        """),
+        Example("""
+        struct ContentView: View {
+            @State ↓private(set) var isPlaying = false
+        """),
+        Example("""
+        struct ContentView: View {
+            @State ↓fileprivate(set) public var isPlaying = false
+        """)
+    ]
+
+    static let corrections: [Example: Example] = [
+        Example("""
+        struct ContentView: View {
+            @State ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct ContentView: View {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct ContentView: View {
+            @State public ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct ContentView: View {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct ContentView: View {
+            @State private(set) ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct ContentView: View {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct ContentView: View {
+            @State private(set) public ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct ContentView: View {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct MyApp: App {
+            @State ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct MyApp: App {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct MyScene: Scene {
+            @State ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct MyScene: Scene {
+                            @State private var isPlaying: Bool = false
+                        }
+                        """),
+        Example("""
+        struct ContentView: View {
+            @State
+            ↓var isPlaying: Bool = false
+        }
+        """): Example("""
+                        struct ContentView: View {
+                            @State
+                            private var isPlaying: Bool = false
+                        }
+                        """)
+    ]
```

Comment:
Please check that comments between the modifiers are kept.

Reply:
Good call, it was dropping leading and trailing trivia from any existing modifiers. 
I updated it so that any trivia attached to the removed modifiers is persisted and appended to the inserted `private` access modifier. 
Can you think of any issues that might arise from that, or other situations that I might need to account for?

Reply:
Looks good now. This might not address all special case that could arise around comments, but that's fine.